docs/community.vmware.vmware_guest_module.rst

```diff
@@ -27,7 +27,7 @@ The below requirements are needed on the host that executes this module.
 
 - python >= 2.6
 - PyVmomi
-
+- You will also need a paid version of ESXi Essentials or a licensed version VMware VCenter. The free version of ESXi is NOT supported.
```

Comment:
This is already documented [here](https://docs.ansible.com/ansible/devel/scenario_guides/vmware_scenarios/faq.html#can-i-deploy-a-virtual-machine-on-a-standalone-esxi-server)

Reply:
Then it should definitely have more visibility. That FAQ entry is no where to be found on the vmware_guest  page. The vmware_guest page should still include this as a requirement, putting it only in the FAQ doesn't make sense. A license is a requirement in this case.